server/subject_transform.go

```diff
@@ -0,0 +1,506 @@
+// Copyright 2018-2023 The NATS Authors
```

Comment:
Is this new or was moved? 

If moved did you use git move?

Reply:
Moved plus modified/added to, did not use git move as it was not a move or rename of a file but instead extracting of just some functions and structs (from accounts.go) to this new file (and then modified/added to) so would not have been able to use git move.